torchvision/transforms/functional.py

```diff
@@ -334,7 +334,7 @@ def to_pil_image(pic, mode=None):
     return Image.fromarray(npimg, mode=mode)
 
 
-def normalize(tensor: Tensor, mean: List[float], std: List[float], inplace: bool = False) -> Tensor:
+def normalize(tensor: Tensor, mean: Union(float, List[float]), std: Union(float, List[float]), inplace: bool = False) -> Tensor:
```

Comment:
If you look at one of the CI jobs below 
![image](https://user-images.githubusercontent.com/1190450/157403431-cde36ec7-5308-475e-816b-336409290c13.png)

you will see that they're failing with the following error:

```
torchvision/transforms/transforms.py:17: in <module>
    from . import functional as F
torchvision/transforms/functional.py:337: in <module>
    def normalize(tensor: Tensor, mean: Union(float, List[float]), std: Union(float, List[float]), inplace: bool = False) -> Tensor:
E   NameError: name 'Union' is not defined
```

So it looks like this is missing an import